reana_workflow_controller/rest.py

```diff
@@ -1234,6 +1233,12 @@ def set_workflow_status(workflow_id_or_name):  # noqa
         if status == START:
             return _start_workflow(workflow,
                                    parameters)
+        elif status == DELETED:
+            all_runs = True if request.json.get('all_runs') else False
```

Comment:
We are missing the documentation of this options on the OpenAPI spec `docstring`.

Reply:
Description added in https://github.com/reanahub/reana-workflow-controller/pull/141/commits/efca3e419311bd5eb2f36c74aa3679a65f3f9e66#diff-3ee9faf1efe690830f3d86cb5d3cf590R1130

Reply:
True, added the schema in commons: https://github.com/reanahub/reana-commons/pull/67